examples/get-started/hook/controls2.js

```diff
@@ -0,0 +1,26 @@
+import {useMap} from 'react-map-gl';
+
+export default function Controls2() {
+  /**
+   * ## This is how `useMap` works:
+   * This component does nothing. It's purpose is to demo `useMap`.
+   * When a component is a child of `<Map>`, `useMap` has a `current` field that references the containing map.
+   * See https://visgl.github.io/react-map-gl/docs/api-reference/use-map
+   * See https://visgl.github.io/react-map-gl/docs/api-reference/types#mapref
+   * ```
+   * const demo = useMap();
+   * console.log('Controls2', {demo});
+   * ```
+   * ### First render:
+   * ```
+   * {
+   *     "demo": {
+   *       "current": {...},
+   *       "mymap": {...}
+   *     }
+   * }
+   * ```
```

Comment:
```suggestion
   * const maps = useMap();
   * console.log('Controls2 useMap()', maps);
   * ```
   * ### First render:
   * ```
   * {
   *     "current": {...}, // this is the same as `mymap`
   *     "mymap": {...}
   * }
   * ```
```